gatsby-config.js

```diff
@@ -20,6 +20,14 @@ module.exports = {
     },
     'gatsby-plugin-react-helmet',
     'gatsby-plugin-glamor',
+    {
+      resolve: 'gatsby-plugin-google-analytics',
+      options: {
+        trackingId: 'GOOGLE_ANALYTICS_TRACKING_ID',
+        head: true,
+        cookieDomain: 'dsmjs.com'
```

Comment:
since we will need to move this site from the current domain to the actual `dsmjs.com` domain once the site is operational, i've been hesitant to do this before we get things migrated. i wouldnt mind having confidence that its ready to go though. 

since i havent configured a new site for GA for a while, do you know off the top of your head if transitioning domains is well supported or if it would be better to wait?

Reply:
Will it hurt to have this config there as a placeholder, then use an actual tracking ID/set it up on Google Analytics website once this goes live on `dsmjs.com`?